lib/transproc/array/combine.rb

```diff
@@ -5,8 +5,9 @@ class Combine
 
       class << self
         def combine(array, mappings)
+          return EMPTY_ARRAY if array.empty?
           root, nodes = array
-          return root if nodes.nil?
+          return root if nodes.nil? || nodes.empty?
```

Comment:
Oh it was just my debug session, `combine` does receive nodes as loaded relation